### PR TITLE
Implement Stack (Python)

### LINKED ISSUE #205: Implement Stack (Python)
## Implement Stack in Python

We need a simple stack implementation written in Python for our repository.

This issue is great for beginners looking to contribute in open source. If you wish to fix this issue please comment that you are working on it. Please read the readme.md and contributing.md files to learn how to contribute.

If you wish to contribute another algorithm/data structure or the same algorithm in a different language, please see if it hasn't been implemented and issue a new request if no issue has been made.

**If you are going to work on the issue yourself, make sure you comment on the issue to let others know.**

### PR DESCRIPTION
**Pull request by asterisk30**
## Description

Added a file named Stack.py

Please link to the issue here:
fixes #205 

## How Has This Been Tested? (Optional)
Tried various output with different data types in linux terminal.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] New Algorithm (non-breaking change which adds functionality)

## Checklist:
Go over all the following points, and put an `x` in all the boxes that apply.

If you're unsure about any of these, don't hesitate to ask. We're here to help!

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by asterisk30** [comment]
i am new to git, and github so accidentally closed the last pull request. 
i have reviewed and changed the suggestion given by you.
please review it again and let me know other changes needed. Thank you

**Comment by GnikDroy** [comment]
@asterisk30 No problem and welcome here.

File: Data Structures/Stack/Python/stack.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    def pop(self):
+        return self.items.pop() 
+ 
+    def disp(self):
+            print(self.items,end=" ")
+ 
+#driver program
+if __name__=='__main__':
+    s = Stack()
+    while True:
```
Comment (GnikDroy) [inline Data Structures/Stack/Python/stack.py]: You can remove this loop.
You just need to demonstrate the stack so a simple program 
```
stack=Stack()
print(stack.is_empty())
stack.push(3)
print(stack.is_empty())
print(stack.pop(3))
```
This is clearer

File: Data Structures/Stack/Python/stack.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * This class Implements Stack
+ * @author asterisk30
+ */
+'''
+class Stack:
+    def __init__(self):
+        self.items = [] 
+ 
+    def is_empty(self):
+        return self.items == []
```
Comment (GnikDroy) [inline Data Structures/Stack/Python/stack.py]: You can replace this with
`return not self.items`
More pythonic.

Comment (GnikDroy) [inline Data Structures/Stack/Python/stack.py]: You can replace this with
`return not self.items`
More pythonic.

File: Data Structures/Stack/Python/stack.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ 
+    def is_empty(self):
+        return self.items == [] 
+    def push(self, data):
+        self.items.append(data) 
+ 
+    def pop(self):
+        return self.items.pop() 
+ 
+    def disp(self):
```
Comment (GnikDroy) [inline Data Structures/Stack/Python/stack.py]: This function is not really required.

**Comment by GnikDroy** [comment]
@asterisk30 There seems to be a merge conflict. Someone has already implemented the stack in python. Can you take on a new issue? You can add a new issue anytime. (Always check if it has already been implemented or already has an issue before creating a new issue)

**Pull request closed without merging**